sway-error/src/error.rs

```diff
@@ -694,6 +694,23 @@ pub enum CompileError {
     CallingPrivateLibraryMethod { name: String, span: Span },
     #[error("Using \"while\" in a predicate is not allowed.")]
     DisallowedWhileInPredicate { span: Span },
+    #[error("Possibly non-zero amount of coins transferred to non-payable contract method \"{fn_name}\".")]
+    PossiblyNonZeroAmountOfCoinsPassedToNonPayableContractMethod { fn_name: Ident, span: Span },
```

Comment:
That's a very long and descriptive error name 😆

Reply:
Lol, fixed in https://github.com/FuelLabs/sway/pull/3450/commits/0e659c633a40f0e56b4350b299b1a8eff9968df0